PyPoE/poe/file/ggpk.py

```diff
@@ -205,10 +205,9 @@ class GGPKRecord(BaseRecord):
 
     @doc(doc=BaseRecord.read)
     def read(self, ggpkfile):
-        # Should be 2, TODO?
-        records = struct.unpack('<i', ggpkfile.read(4))[0]
+        self._container.version = struct.unpack('<i', ggpkfile.read(4))[0]
         self.offsets = []
-        for i in range(0, records):
+        for i in range(2):
```

Comment:
Why has this changed to a constant instead of using the length of `struct.unpack('<i', ggpkfile.read(4))[0]`? Just curious if fixed length will end up biting anyone or anything in the arse?

Reply:
The version field was misunderstood to be a record count in the past as v2 has been around since at least PoE 1.0.0.
Not until v3/v4 with bundles/macOS was it revealed that the `GGPK` chunk only ever has two offsets, one for `FREE` and one for the root `PDIR`